RentACar/RentACar/Helpers/CrudHelper.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+
+namespace RentACar.Utilities
+{
+    static public class CrudHelper
+    {
+        public static Car UpdateCarInListById(List<Car> cars, Car car)
+        {
+            var carUpdated = cars.FirstOrDefault(e => e.Id == car.Id);
+
+            carUpdated.Brand = car.Brand;
+            carUpdated.Color = car.Color;
+            carUpdated.DoorsQuantity = car.DoorsQuantity;
+            carUpdated.Model = car.Model;
+            carUpdated.Transmission = car.Transmission;
+
+            return carUpdated;
+        }
+        public static int GetNewId(List<Car> cars)
+        {
+            if (cars.Count > 0)
+                return cars.Select(x => x.Id).OrderByDescending(x => x).FirstOrDefault() + 1;
```

Comment:
technically speaking this is not _bad_ but I will just drop something here and leave silently
https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.max?view=net-5.0

Reply:
Oh, its more simple now! 🤣  I should take a closer look at the LINQ methods... thanks!
By the way, Can i assume here that the cars in my db (or .json) will always have an id? I was thinking in the case that i have cars without ids.. If id doesn't exist, this method sets id in "0".. I think it is a side effect, but maybe it is better to put the id to "0" instead of null.

Reply:
is up to you whether you allow cars without IDs or not.. is you enforce IDs then you should be good to go. also, if I remember correctly, you can pass default value to the FirstOrDefault() method as a parameter